frontend/src/scenes/dashboard/DashboardReloadAction.tsx

```diff
@@ -72,28 +72,16 @@ export function DashboardReloadAction(): JSX.Element {
                                     <LemonDivider />
                                     <div className="flex flex-col">
                                         <div role="heading" className="text-muted mb-2">
-                                            Refresh interval
+                                            Refresh intervals
                                         </div>
-                                        <Radio.Group
-                                            onChange={(e: RadioChangeEvent) => {
-                                                setAutoRefresh(true, parseInt(e.target.value))
-                                            }}
+                                        <LemonRadio
                                             value={autoRefresh.interval}
-                                            style={{ width: '100%' }}
-                                        >
-                                            <div className="flex flex-col gap-2">
-                                                {intervalOptions.map(({ label, value }) => (
-                                                    <Radio
-                                                        key={value}
-                                                        value={value}
-                                                        style={{ width: '100%' }}
-                                                        disabled={!autoRefresh.enabled}
-                                                    >
-                                                        {label}
-                                                    </Radio>
-                                                ))}
-                                            </div>
-                                        </Radio.Group>
+                                            options={intervalOptions}
+                                            disabled={!autoRefresh.enabled}
```

Comment:
Hmm, it's now possible to both set `disabled` on the whole radio group, as well as on individual items. I think this should work similarly to `LemonSegmentedButton` or the Ant `Radio.Group` – only individual options are disable-able, so that a conflict can't come up.